src/NonLinearEvolutionProblemImplementationBase.cxx

```diff
@@ -244,11 +244,10 @@ namespace mfem_mgis {
 
   bool NonLinearEvolutionProblemImplementationBase::solve(const real t,
                                                           const real dt) {
-    mfem::Vector zero;
     this->setTimeIncrement(dt);
     this->setup(t, dt);
     //    this->computePrediction(t, dt);
-    this->solver->Mult(zero, this->u1);
```

Comment:
This does not change anything with `mfem_mgis::NewtonSolver` (the parameter is ignored) and would not have changed anything with `mfem::NewtonSolver`.

Reply:
Thanks for the clarification. But I was suprised that this `zero` vector was introduced especially, without no specific reason. I think it is better to get rid of it.